examples/rvc-app/rvc-common/pics/RVC Clean Mode Cluster Test Plan.xml

```diff
@@ -0,0 +1,132 @@
+<?xml version="1.0" encoding="utf-8"?><!--
```

Comment:
Why are the TP XMLs added here?

Reply:
To be helpful? I thought these are required to run tests. These are the files from which I generated the txt files. What is there purpose if not to run tests?

Reply:
They're used by the TH, AFAIK. So I wouldn't add them here, we just need the PICS.

Reply:
Yeah, I would just remove these test plan XML things....

Reply:
There is another comment where this is being discussed. I am going to resolve this comment.